### PR TITLE
[Wav2Vec2Conformer] Official release

### PR DESCRIPTION
**Pull request by patrickvonplaten**
# What does this PR do?

Add link to paper and improve readme

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by sgugger** [comment]
Wait, how was the repo-consistency check passing without this? It should have given a big red cross.

**Comment by patrickvonplaten** [comment]
> Wait, how was the repo-consistency check passing without this? It should have given a big red cross.

What is the problem here exactly?

**Comment by patrickvonplaten** [comment]
I see - will do!

**Review by LysandreJik — APPROVED** [review]
:fire:

**Review by sgugger — APPROVED** [review]

File: utils/check_copies.py [hunk 1]
```diff
@@ -485,7 +485,6 @@ def check_model_list_copy(overwrite=False, max_per_line=119):
     "Speech2Text2",
     "Vision Encoder decoder",
     "VisionTextDualEncoder",
-    "Wav2Vec2-Conformer",
```
[the remark this answers is not in the document]
Comment (patrickvonplaten) [reply utils/check_copies.py:488]: Thank you :-)

**Pull request merged**